client/src/constants.ts

```diff
@@ -1 +1 @@
-export const HOSTNAME = '127.0.0.1:3010';
+export const HOSTNAME = '127.0.0.1:3000';
```

Comment:
I forget why we changed this. @erica-zhong do you remember? Does it still work if we change it back?

Reply:
changing to 3001